.github/scripts/label_utils.py

```diff
@@ -24,6 +24,9 @@
 
 If not, please add the `topic: not user facing` label.
```

Comment:
```suggestion
If not, please add the `topic: not user facing` label, either through the GitHub UI or by commenting to pytorchbot, for example
`@pytorchbot label "topic: not user facing"`
```
is more concise/less repetitive

Reply:
I don't want to mention UI, because contributors without write access don't have it and get confused.
Agree on more concise wording, I'll rephrase a bit.